frontend/src/interface/processBackendData.js

```diff
@@ -0,0 +1,53 @@
+import {
+    setMaterialCompositionLabels,
+    setMaterialCompositionData,
+    setImpactAssessmentData,
+    setColumnChartData
+} from 'interface/projectInterface';
+
```

Comment:
Please edit the callback description before commiting this requested change

```suggestion

/**
 * Filters the carbon impact data recieved from API.
 * Filter the Carbon Values of GlobalWarming
 * Maps the Carbon Values and its corresponding life cycle stage.
 * @param data data recieved from PostCalculationRequest
 * @param callback TODO: PLEASE EDIT ME BEFORE COMMITTING!
 */
```

Reply:
I added a few lines, do you think that's enough?